atlasdb-config/src/main/java/com/palantir/atlasdb/http/AtlasDbHttpClients.java

```diff
@@ -56,54 +51,24 @@ private AtlasDbHttpClients() {
             ServerListConfig serverListConfig,
             Class<T> type,
             AuxiliaryRemotingParameters parameters) {
-        return createExperimentallyWithFallback(
-                metricsManager,
-                () -> ConjureJavaRuntimeTargetFactory.DEFAULT.createProxyWithFailover(
-                        serverListConfig, type, parameters),
-                () -> AtlasDbFeignTargetFactory.DEFAULT.createProxyWithFailover(serverListConfig, type, parameters),
-                type,
-                parameters);
+        return VersionSelectingClients.instrumentWithClientVersionTag(
```

Comment:
I haven't read the PR, but we need to make sure we don't miss the code in ExperimentRunningProxy:

```
this.refreshingExperimentalServiceSupplier = Suppliers.memoizeWithExpiration(
                experimentalServiceSupplier::get, CLIENT_REFRESH_INTERVAL.toMinutes(), TimeUnit.MINUTES);  
```

Without this we loose the 30 minute client reload thing, meaning we'll get the lockups back again.